containerfile

```diff
@@ -0,0 +1,36 @@
+FROM python:3.11-slim
+
+WORKDIR /app
+
+# Enable non-interactively
+RUN echo 'debconf debconf/frontend select Noninteractive' | debconf-set-selections
+
+## Update Distro
+RUN apt-get update -y
+
+## Install Dependencies
+RUN apt-get install -y build-essential cmake unzip pkg-config \
+    libjpeg-dev libpng-dev libtiff-dev \
+    libavcodec-dev libavformat-dev libswscale-dev libv4l-dev \
+    libatlas-base-dev gfortran
```

Comment:
Although the installation command seems good enough, I am facing this error on running via docker - 
<img width="1247" alt="image" src="https://github.com/Udayraj123/OMRChecker/assets/16881051/4bce9779-798d-4dc3-a895-fd656917679f">

> qt.qpa.plugin: Could not load the Qt platform plugin "xcb" in "/root/.local/lib/python3.11/site-packages/cv2/qt/plugins" even though it was found.
This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.
>
> Available platform plugins are: xcb.

My template had the CropOnMarkers plugin in it. That's where it's failing
I think you will need to add this
```
export QT_QPA_PLATFORM=offscreen 
```
reference: https://github.com/NVlabs/instant-ngp/discussions/300

Reply:
Tried adding this line in the Dockerfile, but no luck fixing.
```
ENV QT_QPA_PLATFORM offscreen
``` 
Trying the other method of installing `opencv-python-headless` instead of `opencv-python`

Reply:
Installing `opencv-python-headless` didn't help too. Also `libxcb-xinerama0` didn't help.
Tried the below list as well, no luck yet 🤔 
```
'^libxcb.*-dev' libx11-xcb-dev libglu1-mesa-dev libxrender-dev libxi-dev libxkbcommon-dev libxkbcommon-x11-dev
```

Reply:
Oh actually the fix was to use both `opencv-python-headless opencv-contrib-python-headless`, missed it earlier.

Facing [this issue](https://stackoverflow.com/questions/67120450/error-2unspecified-error-the-function-is-not-implemented-rebuild-the-libra) now